<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeightsDialog</class>
 <widget class="QDialog" name="WeightsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weights</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="idLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID Field</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="idFieldCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="mainTab">
     <widget class="QWidget" name="contiguityTab">
      <attribute name="title">
       <string>Contiguity</string>
      </attribute>
      <layout class="QVBoxLayout" name="contigLayout">
       <item>
        <widget class="QGroupBox" name="contiguityGroup">
         <property name="title">
          <string>Contiguity Method</string>
         </property>
         <layout class="QFormLayout" name="contiguityMethodLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="contiguityLabel">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="contiguityCombo">
            <item>
             <property name="text">
              <string>Queen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rook</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Block</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="blockLabel">
            <property name="text">
             <string>Block column</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="blockColumnCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="contiguityParamsGroup">
         <property name="title">
          <string>Parameters</string>
         </property>
         <layout class="QFormLayout" name="contiguityParamsLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="precisionLabel">
            <property name="text">
             <string>Precision threshold</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="precisionSpin">
            <property name="minimum">
             <double>0.0</double>
            </property>
            <property name="maximum">
             <double>1000000.0</double>
            </property>
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="orderLabel">
            <property name="text">
             <string>Order of contiguity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="orderSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="includeLowerCheck">
            <property name="text">
             <string>Include lower orders</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="distanceTab">
      <attribute name="title">
       <string>Distance</string>
      </attribute>
      <layout class="QVBoxLayout" name="distanceLayout">
       <item>
        <widget class="QTabWidget" name="distanceTypeTabs">
         <widget class="QWidget" name="geomDistanceTab">
          <attribute name="title">
           <string>Geometric Centroid</string>
          </attribute>
          <layout class="QVBoxLayout" name="geomLayout">
           <item>
            <layout class="QFormLayout" name="coordLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="xCoordLabel">
               <property name="text">
                <string>X coordinate</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="xCoordCombo"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="yCoordLabel">
               <property name="text">
                <string>Y coordinate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="yCoordCombo"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="geomMetricLabel">
               <property name="text">
                <string>Distance metric</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="geomMetricCombo">
               <item>
                <property name="text">
                 <string>Euclidean</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arc Distance</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="geomMethodTabs">
             <widget class="QWidget" name="bandTab">
              <attribute name="title">
               <string>Distance Band</string>
              </attribute>
              <layout class="QFormLayout" name="bandLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="bandwidthLabel">
                 <property name="text">
                  <string>Bandwidth</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="bandwidthSpin">
                 <property name="minimum">
                  <double>0.0</double>
                 </property>
                 <property name="maximum">
                  <double>1000000.0</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QSlider" name="bandwidthSlider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="bandInverseCheck">
                 <property name="text">
                  <string>Use inverse distance</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="bandPowerLabel">
                 <property name="text">
                  <string>Power</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="bandPowerSpin">
                 <property name="minimum">
                  <double>0.0</double>
                 </property>
                 <property name="maximum">
                  <double>10.0</double>
                 </property>
                 <property name="singleStep">
                  <double>0.1</double>
                 </property>
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="knnTab">
              <attribute name="title">
               <string>K-nearest neighbors</string>
              </attribute>
              <layout class="QFormLayout" name="knnLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="neighborsLabel">
                 <property name="text">
                  <string>Number of neighbors</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="knnSpin">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="knnInverseCheck">
                 <property name="text">
                  <string>Use inverse distance</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="knnPowerLabel">
                 <property name="text">
                  <string>Power</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="knnPowerSpin">
                 <property name="minimum">
                  <double>0.0</double>
                 </property>
                 <property name="maximum">
                  <double>10.0</double>
                 </property>
                 <property name="singleStep">
                  <double>0.1</double>
                 </property>
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="kernelTab">
              <attribute name="title">
               <string>Kernel</string>
              </attribute>
              <layout class="QVBoxLayout" name="kernelLayout">
               <item>
                <layout class="QFormLayout" name="kernelTopLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="kernelFuncLabel">
                   <property name="text">
                    <string>Kernel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="kernelFunctionCombo">
                   <item>
                    <property name="text">
                     <string>Gaussian</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Quartic</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Epanechnikov</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Triangular</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Uniform</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="diagOptionLabel">
                   <property name="text">
                    <string>Diagonal treatment</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="diagOptionCombo">
                   <item>
                    <property name="text">
                     <string>Diagonal weight = 1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Apply kernel to diagonal</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="bandwidthGroup">
                 <property name="title">
                  <string>Bandwidth</string>
                 </property>
                 <layout class="QVBoxLayout" name="bandwidthGroupLayout">
                  <item>
                   <widget class="QRadioButton" name="bandwidthValueRadio">
                    <property name="text">
                     <string>Fixed value</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QFormLayout" name="fixedBandwidthLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="kernelBandwidthLabel">
                      <property name="text">
                       <string>Bandwidth</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QDoubleSpinBox" name="kernelBandwidthSpin">
                      <property name="minimum">
                       <double>0.0</double>
                      </property>
                      <property name="maximum">
                       <double>1000000.0</double>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="2">
                     <widget class="QSlider" name="kernelBandwidthSlider">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="adaptiveBandwidthRadio">
                    <property name="text">
                     <string>Adaptive bandwidth</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QFormLayout" name="adaptiveLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="kernelNeighborsLabel">
                      <property name="text">
                       <string>Number of neighbors</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QSpinBox" name="kernelNeighborsSpin">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="maxKnnRadio">
                    <property name="text">
                     <string>Use max knn distance</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="variablesTab">
          <attribute name="title">
           <string>Variables</string>
          </attribute>
          <layout class="QFormLayout" name="variablesLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="variablesLabel">
             <property name="text">
              <string>Variables</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="variablesEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="transformLabel">
             <property name="text">
              <string>Transformation</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="transformCombo">
             <item>
              <property name="text">
               <string>Standardize (Z)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Standardize (MAD)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Demean</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Raw</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="varMetricLabel">
             <property name="text">
              <string>Distance metric</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="varMetricCombo">
             <item>
              <property name="text">
               <string>Euclidean</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Manhattan</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>